success-stories/index.html

```diff
@@ -91,9 +91,9 @@ <h6>Conference talks, interviews, and video testimonials from the Spinnaker Comm
     </div></div>
 
     <div class="video-item">
-    <p>Andy Glover: How Netflix Runs Spinnaker @ Scale</p>
+    <p>Adobe Deploys Critical Apps With Spinnaker </p>
```

Comment:
@dnilasor , replacing Andy's video rather than adding?

Reply:
pre-site-refresh, yes. I thought the SIG agreed we don't want too many videos on there, and I am not able to design/implement a better videos area on this page for now. We need multiples of 3 for what I hacked together to work. Netflix has a prominent use case on the page already and I think for this page, adding companies that aren't clearly associated with Spinnaker success already is a priority. 

@dorbin open to alternatives or to removing a different video.

Reply:
👍 SGTM!